### ISSUE TITLE
Contrast ratio test fails on background gradients

### ISSUE THREAD

**Issue by matteobelfiore** [addyosmani/a11y#67]
When the test runs on pages with a gradient as a background, the background is evaluated as `#ffffff`. If the text colour happens to be too light —or white as well— the test obviously fails with:

```
Text elements should have a reasonable contrast ratio
```
It would probably be a good idea to pick the lightest of the gradient colours and run the test on that one instead, or on each colour separately.
